qiskit/schemas/backend_configuration_schema.json

```diff
@@ -157,6 +157,11 @@
                     "description": "Whether delay between programs can be set dynamically using 'rep_delay').",
                     "default": false
                     },
+                "measure_esp_enabled": {
```

Comment:
Version bump here as well.

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6543/commits/de6d6073315d5607ef59c929861b38aa6d47847a.